deps/github.com/openshift/kubernetes/openshift-hack/rebase.sh

```
@@ -125,51 +124,64 @@ fi
+if command -v gh &>/dev/null; then
+  XY=$(echo "$k8s_tag" | sed -E "s/v(1\.[0-9]+)\.[0-9]+/\1/")
+  ver=$(echo "$k8s_tag" | sed "s/\.//g")
+  link="https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-$XY.md#$ver"
 
+  title="Rebase $k8s_tag in $openshift_release"
+  if [ -n "${jira_id}" ]; then
+    title="$jira_id: $title"
   fi
+
+  gh pr create \
+    --title "$title" \
```

Shellcheck (0.11.0)

[style] Prefer putting braces around variable references even when not strictly required.

(SC2250)